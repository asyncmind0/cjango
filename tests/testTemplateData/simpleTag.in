{% if is_logged_in %}Thanks for logging in!{% else %}Please log in.{% endif %}
