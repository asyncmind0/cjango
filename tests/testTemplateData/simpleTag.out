Thanks for logging in!Please log in.
